Integrations/Gmail/Gmail.py

```diff
@@ -315,9 +421,56 @@ def users_to_entry(title, response):
             'Gmail': {'Address': user_data.get('primaryEmail')},
             'Group': user_data.get('kind'),
             'CustomerId': user_data.get('customerId'),
+            'Groups': user_data.get('kind'),
+            'Domain': user_data.get('primaryEmail').split('@')[1],
+            'Username': (user_data.get('name').get('givenName')
+                         if user_data.get('name') and 'givenName' in user_data.get('name') else None),
+            'OrganizationUnit': (user_data.get('organizations')[0].get('name')
+                                 if (user_data.get('organizations') and len(user_data.get('organizations')) > 0
+                                     and 'name' in user_data.get('organizations')[0]) else None)
+
+        })
+    headers = ['Type', 'ID', 'UserName',
+               'DisplayName', 'Email', 'Group', 'CustomerId', 'Email',
+               'Groups', 'Domain', 'Username', 'OrganizationUnit']
+
+    return {
+        'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents': context,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(title, context, headers),
+        'EntryContext': {'Account(val.ID && val.Type && val.ID == obj.ID && val.Type == obj.Type)': context}
+    }
+
+
+def hide_users_to_entry(title, response):
+    context = []
+    for user_data in response:
+        context.append({
+            'Type': 'Google',
+            'ID': user_data.get('id'),
+            'UserName': (user_data.get('name').get('givenName')
+                         if user_data.get('name') and 'givenName' in user_data.get('name') else None),
+            'DisplayName': (user_data.get('name').get('fullName')
+                            if user_data.get('name') and 'fullName' in user_data.get('name') else None),
+            'Email': {'Address': user_data.get('primaryEmail')},
+            'Gmail': {'Address': user_data.get('primaryEmail')},
+            'Group': user_data.get('kind'),
+            'CustomerId': user_data.get('customerId'),
+            'VisibleInDirectory': user_data.get('includeInGlobalAddressList'),
+            'Groups': user_data.get('kind'),
+            'Domain': user_data.get('primaryEmail').split('@')[1],
+            'Username': (user_data.get('name').get('givenName')
+                         if user_data.get('name') and 'givenName' in user_data.get('name') else None),
+            'OrganizationUnit': (user_data.get('organizations')[0].get('name')
+                                 if (user_data.get('organizations') and len(user_data.get('organizations')) > 0
+                                     and 'name' in user_data.get('organizations')[0]) else None)
+
         })
     headers = ['Type', 'ID', 'UserName',
-               'DisplayName', 'Email', 'Group', 'CustomerId']
+               'DisplayName', 'Email', 'Group', 'CustomerId', 'Visible In Directory', 'CustomerId',
```

Comment:
'Visible In Directory' does not appear in the dictionary and will always be empty, use the `headerTransform` in the `tableToMarkdown` in order to use the spacing.